test/e2e/lib/media-helper.js

```diff
@@ -69,29 +69,27 @@ export function deleteFile( fileDetails ) {
 	return fs.deleteSync( fileDetails.file );
 }
 
+export const screenshotsDir = path.resolve(
+	process.env.TEMP_ASSET_PATH || __dirname + '/..',
```

Comment:
I think this could be problematic and I'll need to revert this change. The __dirname was contextual to the script calling the `writeScreenshot` func, and now it runs once when the module is loaded.

Reply:
I've [tested](https://github.com/fullofcaffeine/dirnameexperiment) this, and it doesn't seem to matter if `__dirname` is indirectly called from another module/script through a function,  it will always return the `__dirname` to the module/script it's in (even if lazily called by a function), so I think it's fine to leave this piece of code like this.